README.md

```diff
@@ -101,27 +99,29 @@ OpenAPI linter like [Spectral](https://stoplight.io/open-source/spectral) can co
 
 ## Mocking
 
-A fake server that takes a description document as input, then routes incoming HTTP requests to example responses or dynamically generates examples. 
+A fake server that takes a description document as input, then routes incoming HTTP requests to example responses or dynamically generates examples.
 
 _A list of mock servers is available on [OpenAPI.Tools](https://openapi.tools/#mock-servers)._
 
 ## OAS
 
-This is just shorthand for the OpenAPI Specification, which is just [Markdown files on the internet](https://github.com/OAI/OpenAPI-Specification/tree/master/versions) defining how each version of OpenAPI should work.
+This is just shorthand for the [OpenAPI Specification[(#openapi)], which is [Markdown files on the internet](https://github.com/OAI/OpenAPI-Specification/tree/master/versions) and [JSON schemas](https://github.com/OAI/OpenAPI-Specification/tree/master/schemas) defining how each version of OpenAPI should work.
```

Comment:
Let's not advertise these too much, there has been increasing concern from the TSC that these schemas were just meant to be advisory but many people have started to think they're actually part of the spec. They are not, and this sentence suggests they are.